lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -1690,3 +1760,24 @@ int parse_periodic_search_pattern(const char *const pattern_str,
 
 	return 0;
 }
+
+int lte_lc_rai_set(void)
+{
+	int err;
+
+	LOG_DBG("Enabling RAI with notifications");
```

Comment:
This log is misleading when `CONFIG_LTE_RAI_REQ` is disabled. Maybe change it to something like "Configuring RAI". If `%RAI=2` fails, there will anyway be a separate log for that.

Reply:
I've made it an if-else-statement so we clearly see in the boot how it's configured:
```
	if (IS_ENABLED(CONFIG_LTE_RAI_REQ)) {
		LOG_DBG("Enabling RAI with notifications");
	} else {
		LOG_DBG("Disabling RAI");
	}
```